lib/interactive.dart

```diff
@@ -126,6 +133,35 @@ class Interactive {
     return color;
   }
 
+  String _lookupFilterLabels(String filter, String option) {
+    String label = "Unknown";
+    for (var f in _filters) {
+      if (f.value == filter) {
+        for (var o in f.options) {
+          if (o.value == option) {
+            label = o.label;
+            continue;
```

Comment:
Can we return here instead of continuing?

Reply:
fixed